nav2_planner_metrics/nav2_planner_metrics/metrics.py

```diff
@@ -0,0 +1,157 @@
+#! /usr/bin/env python3
```

Comment:
I don't want to put this into a new package unless we're planning on doing alot more here in the future, which I don't think either of us are. I think this should go into `/tools` https://github.com/ros-planning/navigation2/tree/main/tools